tests/model_fields/tests.py

```diff
@@ -271,6 +271,14 @@ class Meta:
 
         self.assertEqual(ConcreteReferent._meta.get_field('reference').related_model, Refered)
 
+    @test.skipUnlessDBFeature('can_defer_constraint_checks')
+    def test_defers_constraint_check(self):
+        b = Bar.objects.create(b='abc', a_id=5)
+        self.assertEqual(b.a_id, 5)
+        # Delete the object to avoid the error once constraints are
+        # checked.
+        b.delete()
```

Comment:
No, this is not a regression test. I didn't intend it to be. I wanted to use a very straightforward test to both demonstrate to myself:

* Without my change, constraint violations would not cause an error in a test
* With my change, constraint violations would cause an error in a test

I added the `b.delete()` after the fact so the test would pass. If this isn't considered a useful test by others, I can delete it.